spring-cloud-dataflow-module-deployers/spring-cloud-dataflow-module-deployer-kubernetes/README.md

```diff
@@ -0,0 +1,176 @@
+SPI implementation for deploying Spring Cloud Stream modules to [Kubernetes](http://http://kubernetes.io/).
+
+### Setup a Kubernetes Cluster 
+
+If you don't have access to a Kubernetes Cluster, you can deploy one locally.
+This requires a local functional Docker installation and the usage of some 
+nice scripts to setup [Kubernetes](https://github.com/olmoser/docker-compose-kubernetes).
+
+In a nutshell, do the following steps:
+```
+$ git clone https://github.com/olmoser/docker-compose-kubernetes
+$ cd docker-compose-kubernetes
+$ docker-machine create --driver virtualbox sc-dataflow-dev 
+$ docker-machine start sc-dataflow-dev 
+$ eval "$(docker-machine env sc-dataflow-dev)"
+$ ./kube-up.sh
+```
+
+Once the cluster is up, configure you local system to your local DNS server `10.0.0.10` 
+as the main resolver. On Mac, you can create a network profile for easy switching.
+
+After installing a cluster, install the `kubectl` binary and ensure you have proper access to the
+cluster. 
+
+
+### Run the Kubernetes Deployer
+
+1. Checkout the code:
+
+   ```
+   $ git clone https://github.com/frosenberg/spring-cloud-dataflow
+   $ cd spring-cloud-dataflow
+   ```
+
+1. Start the Kubernetes infrastructure as described above.
+
+1. Start the Redis server within your Kubernetes cluster:
+
+   ```
+   $ kubectl create -f src/main/kubernetes/redis-controller.yml
```

Comment:
Not sure `src/main/kubernetes` is the best place for those kind of files. Will need to have a look.